modify_record.js

```diff
@@ -25,40 +25,85 @@ function save_row(id) {
     var email = document.getElementById("email_text" + id).value;
     var gender = document.getElementById("gender_text" + id).value;
     var occupation = document.getElementById("occu_text" + id).value;
-     console.log("ajaxhdfvb");
-    $.ajax
-        ({
-            type: 'post',
-            url: 'modifyRecord.php',
-            data: {
-                edit_row: 'edit_row',
-                row_id: id,
-                fname_val: fname,
-                lname_val: lname,
-                age_val: age,
-                email_val: email,
-                gender_val: gender,
-                occu_val: occupation    
-            },
-            success: function (response) {
-                if (response == "success") {
-                    document.getElementById("fname_val" + id).innerHTML = fname;
-                    document.getElementById("lname_val" + id).innerHTML = lname;
-                    document.getElementById("age_val" + id).innerHTML = age;
-                    document.getElementById("email_val" + id).innerHTML = email;
-                    document.getElementById("gender_val" + id).innerHTML = gender;
-                    document.getElementById("occu_val" + id).innerHTML = occupation;
 
-                    document.getElementById("edit_button" + id).style.display = "block";
-                    document.getElementById("save_button" + id).style.display = "none";
-                    alert("This row has been edited successfully");
+
+    function validate() {
```

Comment:
do not write this type of code. change it. Never decare a fucntion insisde another function. Its a very bad practice